Lecture6(GithubAPI)/Component/SearchComponent.js

```diff
@@ -23,7 +23,7 @@ const SearchComponent = ({ DataFromAPI, setFilteredData, setNoRecord }) => {
       onSubmit={(e) => {
         e.preventDefault();
         const filteredData = searchTeamdetails(searchText, DataFromAPI);
-        console.info(filteredData);
+        // console.info(filteredData);
         if (filteredData.length > 0) {
           setFilteredData(filteredData);
```

Comment:
What would be value of filteredData (as in state) then when filteredData length is 0?

Reply:
it will be [] empty array when searched with anything which don't matches.